### ISSUE TITLE
Upgrade manifest.json v2 to v3

### ISSUE THREAD

**Issue by dhimanAbhi** [BinaryMuse/chrome-fast-tab-switcher#25]
The current chrome extension is built on v2 of manifest.json. It must be upgraded to v3.

![Issue](https://user-images.githubusercontent.com/87617110/190629520-e4919174-5100-4b1d-aae0-95ffdef0fe4e.png)

**Comment by eyahlin** [comment]
Yes, I second this. It no longer appears in the Chrome extension list because of it.
